Xamarin.Forms.Core/Items/CarouselView.cs

```diff
@@ -216,11 +216,13 @@ static int GetPositionForItem(CarouselView carouselView, object item)
 			return 0;
 		}
 
+		[EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
require a `using`

Reply:
Ops, it seems that I made commit too fast and git didn't detect the change. Added!